### PR TITLE
[WIP] Fixes to docs files for MDX parser

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

The Markdown/MDX parser that we're using to build the new documentation site was throwing errors on these two files for some bad Markdown syntax. I fixed those, and while I was in there, I fixed a few linter errors in both files.

With these changes merged, we'll be able to pull all the Markdown files from the repo directly into the documentation site.

##### Component Name

docs

### CODE REVIEW

**Comment by joelhans** [comment]
Changing to WIP since I'm finding more now that I'm actually trying to build the Gatsby site on Netlify to test. Will update with more fixes.

File: collectors/python.d.plugin/uwsgi/README.md [hunk 1]
```diff
@@ -36,6 +32,6 @@ localhost:
   port     : 1717
 ```
 
-When no configuration file is found, module tries to connect to TCP/IP socket: `localhost:1717`.
+If Netdata can't find a configuration file, the module tries to connect to thde TCP/IP socket at `localhost:1717`.
```
Comment (ilyam8) [inline collectors/python.d.plugin/uwsgi/README.md]: > thde

a typo

Comment (ilyam8) [reply collectors/python.d.plugin/uwsgi/README.md]: > If Netdata can't find a configuration file

> collectors/python.d.plugin/uwsgi/README.md

Technically it is not Netdata. `python.d.plugin` (external plugin) finds config files.

**Comment by joelhans** [comment]
Closing in favor of a simplified version at #6877.

**Pull request closed without merging**